compiler/fir/analysis-tests/testData/resolveWithStdlib/callableReferences/ifWithCR.kt

```diff
@@ -5,4 +5,4 @@ fun main(x: Int) {
     val x = if (x > 1) x::readOnlyWrapper else x::mutableWrapper
 
     x.get()
-}
+}
```

Comment:
What is this change?

Reply:
This change should not be here. Please revert it (e.g. "Revert selected changes" from Git plugin).